sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -363,6 +391,7 @@ public ConfigurationClientBuilder addPolicy(HttpPipelinePolicy policy) {
      * @param client The HTTP client to use for requests.
      * @return The updated ConfigurationClientBuilder object.
      */
+    @Override
```

Comment:
As part of this change, we should also update the javadocs of all overridden trait methods to use {@inheritdoc} so that we can standardize the docs at the trait interface level.

Reply:
While attempting this we've run into an issue that inhertidoc doesn't work cross-package as expected. This will get addressed later when tooling is fixed. https://github.com/Azure/azure-sdk-for-java/issues/26830
For now I'll copy docs from interfaces...